lib/Utility/EncryptionUtil.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:encrypt/encrypt.dart';
+
+
+class EncryptUtil{
+
+ /// Encryption key to use in encrypting and decrypting text notes
+  static final _ENCRYPTION_KEY = 'b4CplJtMrcfEg7LJhL2dZyEwgvHP/75w';
```

Comment:
How can we store this secretly ?

Reply:
I found this in the mix of the code that was borrowed from the other team - we definitely won't be using it like this.